Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -49,13 +69,11 @@ Check for files and directories typically associated with jailbreaks, such as:
 /var/log/syslog
 ```
 
-#### Checking File Permissions
-
-Another way to check for jailbreaking mechanisms is to try to write to a location that's outside the application's sandbox. You can do this by having the application attempt to create a file in, for example, the `/private directory`. If the file is created successfully, the device has been jailbroken.
+**Checking File Permissions:**
 
-**Swift:**
+The app might be trying to write to a location that's outside the application's sandbox. For instance, it may attempt to create a file in, for example, the `/private directory`. If the file is created successfully, the app  might assume that the device has been jailbroken.
```

Comment:
```suggestion
The app might be trying to write to a location that's outside the application's sandbox. For instance, it may attempt to create a file in, for example, the `/private` directory. If the file is created successfully, the app can assume that the device has been jailbroken.
```